web/ce/components/workspace/delete-workspace.tsx

```diff
@@ -0,0 +1,65 @@
+import { FC, useState } from "react";
+import { observer } from "mobx-react";
+import { ChevronDown, ChevronUp } from "lucide-react";
+import { Disclosure, Transition } from "@headlessui/react";
+// types
+import { IWorkspace } from "@plane/types";
+// ui
+import { Button } from "@plane/ui";
+// components
+import { DeleteWorkspaceModal } from "@/components/workspace";
+
+type TDeleteWorkspace = {
+  workspace: IWorkspace | null;
+};
+
+export const DeleteWorkspace: FC<TDeleteWorkspace> = observer((props) => {
+  const { workspace } = props;
+  // states
+  const [deleteWorkspaceModal, setDeleteWorkspaceModal] = useState(false);
+
+  return (
+    <>
+      <DeleteWorkspaceModal
+        data={workspace}
+        isOpen={deleteWorkspaceModal}
+        onClose={() => setDeleteWorkspaceModal(false)}
+      />
+      <Disclosure as="div" className="border-t border-custom-border-100">
```

Comment:
Can you use the collapsible component here.

Reply:
I have added that change.